CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 29-cvs (in development)
 =======================
 
+- New features:
```

Comment:
The changelog's been updated meanwhile.  Please rebase on master and merge the changes.